packages/create-cloudflare/src/cli.ts

```diff
@@ -30,6 +32,39 @@ const WRANGLER_DEFAULTS = {
 export const main = async (argv: string[]) => {
 	const args = await parseArgs(argv);
 
+	// Print a newline
+	logRaw("");
+
+	if (args.autoUpdate && (await isUpdateAvailable())) {
+		await runLatest();
+	} else {
+		await runCli(args);
+	}
+};
+
+const isUpdateAvailable = async () => {
+	if (process.env.VITEST) {
```

Comment:
are you hardcoding this to make our own e2e test suite work? could we do something more generic like `CI` or use both VITEST and CI?

CI use used by almost everyone: https://github.com/orgs/community/discussions/26628

Reply:
I've had the same thought but I've been holding off until I had a good first use case for "CI" -- meaning something I only want to happen in github actions but not on dev laptops.

For example we'd want skip auto-updates when running the tests locally, and by checking VITEST which gets set automatically by the runner, devs can't make a mistake of forgetting to set CI in a test run and seeing inconsistent results.